Build failure: cadence

### Steps To Reproduce

Steps to reproduce the behavior:
1. build cadence

### Build log

```
error: builder for '/nix/store/mivzc7yny51qw00s9hvx1wxvjmn0bh50-jack_capture-0.9.73.drv' failed with exit code 2;
       last 30 log lines:
       > unpacking sources
       > unpacking source archive /nix/store/fvwlxs27nksacab2sllw2nxbhj6fm6fb-jack_capture-0.9.73.tar.gz
       > source root is jack_capture-0.9.73
       > setting SOURCE_DATE_EPOCH to timestamp 1495191360 of file jack_capture-0.9.73/vringbuffer.h
       > patching sources
       > updateAutotoolsGnuConfigScriptsPhase
       > configuring
       > no configure script, doing nothing
       > building
       > bash gen_setformat_c.sh >setformat.c
       > bash gen_das_config_h.sh >das_config.h
       >
       > jack_capture was configured with the following options:
       > ------------------------------
       > #define HAVE_OGG 1
       > #define HAVE_LAME 0
       > #define HAVE_LIBLO 0
       > #define NEW_JACK_LATENCY_API 1
       > ------------------------------
       >
       > cat das_config.h |grep COMPILEFLAGS|sed s/\\/\\/COMPILEFLAGS//|tr '\n' ' ' >config_flags
       > gcc -I/opt/local/include -O3 -DVERSION=\"0.9.73\" -Wall -Wextra -Wno-unused jack_capture.c vringbuffer.c upwaker.c osc.c -o jack_capture  -ljack -lsndfile -lm -lpthread -lrt `cat config_flags`
       > jack_capture.c:912:5: error: conflicting types for 'asprintf'; have 'int(char **, char *, ...)'
       >   912 | int asprintf(char **buffer, char *fmt, ...) {
       >       |     ^~~~~~~~
       > In file included from jack_capture.c:24:
       > /nix/store/x8lqlydsxbrwvf6p7v18gws8kn1xl37f-glibc-2.38-23-dev/include/stdio.h:401:12: note: previous declaration of 'asprintf' with type 'int(char ** restrict,  const char * restrict, ...)'
       >   401 | extern int asprintf (char **__restrict __ptr,
       >       |            ^~~~~~~~
       > make: *** [Makefile:63: jack_capture] Error 1
       For full logs, run 'nix log /nix/store/mivzc7yny51qw00s9hvx1wxvjmn0bh50-jack_capture-0.9.73.drv'.
error: 1 dependencies of derivation '/nix/store/2ll21bhv3v6b5bskm5xcrzyb9cjpj51k-cadence-0.9.1.drv' failed to build

```

### Additional context

Add any other context about the problem here.

### Notify maintainers

<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
Welcome to khanelinix
 - system: `"x86_64-linux"`
 - host os: `Linux 6.5.7, NixOS, 23.11 (Tapir), 23.11.20231019.7c9cc5a`
 - multi-user?: `yes`
 - sandbox: `relaxed`
 - version: `nix-env (Nix) 2.18.1`
 - channels(root): `""`
 - nixpkgs: `/etc/nix/inputs/nixpkgs`
```

--- a/pkgs/applications/audio/jack-capture/default.nix
+++ b/pkgs/applications/audio/jack-capture/default.nix
@@ -1,12 +1,14 @@
-{ lib, stdenv, fetchurl, libjack2, libsndfile, pkg-config }:
+{ lib, stdenv, fetchFromGitHub, libjack2, libsndfile, pkg-config }:
 
 stdenv.mkDerivation rec {
   pname = "jack_capture";
-  version = "0.9.73";
+  version = "0.9.73.2023-01-04";
 
-  src = fetchurl {
-    url = "https://archive.notam02.no/arkiv/src/${pname}-${version}.tar.gz";
-    sha256 = "1pji0zdwm3kxjrkbzj7fnxhr8ncrc8pyqnwyrh47fhypgqjv1br1";
+  src = fetchFromGitHub {
+    owner = "kmatheussen";
+    repo = "jack_capture";
+    rev = "a539d444d388c4cfed7279e385830e7767d59c41";
+    sha256 = "sha256-2DavZS4esV17a3vkiPvfCfp0QF94ZcXqdIw84h9HDjA=";
   };
 
   nativeBuildInputs = [ pkg-config ];
@@ -23,9 +25,9 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
     description = "A program for recording soundfiles with jack";
-    homepage = "http://archive.notam02.no/arkiv/src";
+    homepage = "https://github.com/kmatheussen/jack_capture/";
     license = licenses.gpl2;
-    maintainers = [ maintainers.goibhniu ];
+    maintainers = with maintainers; [ goibhniu orivej ];
     platforms = lib.platforms.linux;
   };
 }

